pkgs/servers/sql/postgresql/ext/citus.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, autoreconfHook
+, postgresql
+, curl
+, fetchFromGitHub
+, lz4
+, zstd
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "citus";
+  version = "10.0.3";
+
+  src = fetchFromGitHub {
+    owner = "citusdata";
+    repo = "citus";
+    rev = "v${version}";
+    sha256 = "hzR4tQnws5kYyLY8xzoIiOEJrCQFj0t2d+lJg2c8/AE=";
+    fetchSubmodules = false;
+  };
+
+  buildInputs = [
+    postgresql
+    curl
+    lz4
+    zstd.out
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    zstd.dev
+  ];
+
+  # dirty hack to give us a clean install tree
+  installPhase = ''
+    make install DESTDIR=/tmp/citus
+    mkdir -p $out
+    mv /tmp/citus/nix/store/*/* $out
```

Comment:
This looks really hacky. Also this probably breaks when building two times the package in parallel. It must at least use $TMP or mktemp -d.

Reply:
Another way is to change the postgres pkgconfig paths, but I don't know any way to override those.

Reply:
substituteInPlace maybe?

Reply:
The problem is that it's referencing some makefile from postgresql itself (think basically like building a kernel module).

Reply:
I think you can use the expression of the previous attempt https://github.com/NixOS/nixpkgs/pull/38698/commits/4119c9fde040aa0a938974950ad65318bfa4bf16